q2_cutadapt/tests/test_demux.py

```diff
@@ -200,6 +200,27 @@ def test_batch_size_odd_number_of_samples(self):
         # obs_untrimmed should be empty, since everything matched
         self.assert_untrimmed_results(b'', obs_untrimmed_art)
 
+    def test_min_length(self):
+        metadata = CategoricalMetadataColumn(
+            # The third barcode is meant to completely remove the only GGGG
+            # coded sequence
+            pd.Series(['AAAA','CCCC','GGGGACGTACGT'], name='Barcode',
+                      index=pd.Index(['sample_a', 'sample_b', 'sample_c'],
+                      name='id')))
+
+        with redirected_stdio(stderr=os.devnull):
+            obs_demuxed_art, obs_untrimmed_art = \
+                self.demux_single_fn(self.muxed_sequences, metadata,
+                                     batch_size=2)
```

Comment:
We don't need to set the `batch_size` explicitly here, I suspect this is left-over from another test (?)

Reply:
Yep, left over from another test.